lib/record.js

```diff
@@ -38,6 +39,23 @@ function createRecord({ files, config }) {
   };
 }
 
+function unifyFilePaths(record) {
+  if (record === null) {
+    return record;
+  }
+
+  return {
+    ...record,
```

Comment:
Eslint is complaining about this spread. Can you change the `ecmaVersion` in `.eslintrc.js` to `2018`?

Reply:
Done